src/main/java/uk/gov/di/ipv/cri/common/library/service/SessionService.java

```diff
@@ -104,12 +104,20 @@ public SessionItem validateSessionId(String sessionId)
     private void setSessionItemsToLogging(SessionItem sessionItem) {
         Optional.ofNullable(sessionItem)
                 .ifPresent(
-                        s ->
-                                Optional.ofNullable(s.getClientSessionId())
-                                        .ifPresent(
-                                                id ->
-                                                        LoggingUtils.appendKey(
-                                                                GOVUK_SIGNIN_JOURNEY_ID, id)));
+                        s -> {
+                            Optional.ofNullable(s.getClientSessionId())
+                                    .ifPresent(
+                                            id ->
+                                                    LoggingUtils.appendKey(
+                                                            GOVUK_SIGNIN_JOURNEY_ID, id));
+                            Optional.ofNullable(s.getEvidenceRequest())
+                                    .ifPresent(
+                                            ev ->
+                                                    LoggingUtils.appendKey(
```

Comment:
Is this missing a test check the service test there probably for verify for the GOVUK_SIGNIN_JOURNEY_ID one

Reply:
I think we should call `validateSessionId` in all lambda(s) in Experian KBV it is currently only called in `questionHandler` and `questionAnswerHandler`

Reply:
Might also need to do something here https://github.com/govuk-one-login/ipv-cri-common-lambdas/blob/main/lambdas/src/middlewares/session/set-gov-uk-signing-journey-id-middleware.ts

Reply:
On `validateSessionId` 
- Abandon lambda does not call this although it does set the logging keys here: https://github.com/govuk-one-login/ipv-cri-lib/blob/main/src/main/java/uk/gov/di/ipv/cri/common/library/service/SessionService.java#L117
- Issue Credential does call it through: https://github.com/govuk-one-login/ipv-cri-lib/blob/main/src/main/java/uk/gov/di/ipv/cri/common/library/service/SessionService.java#L141

Is there a reason you can think why Abandon does not, looks intentional?

Reply:
On Common-lambdas - Yep good spot, I hadn't thought of adding it to those lambdas just to KBV ones. Draft PR up here: https://github.com/govuk-one-login/ipv-cri-common-lambdas/pull/317

Reply:
I don't think there is, it think it was just missed

Reply:
Ok I have updated that on my branch for kbv-api